stock_pack_operation_auto_fill/__openerp__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Stock Pack Operation Auto Fill',
+    'summary': """
+        Stock pack operation auto fill""",
+    'version': '9.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu/',
```

Comment:
In website should be https://github.com/OCA/stock-logistics-workflow according to https://github.com/OCA/maintainer-tools/blob/f036ab8c259298769926fd98bad0c7e4fe980608/template/module/__openerp__.py#L9